quickstarts/activemq/config.yml

```diff
@@ -0,0 +1,43 @@
+
+slug: activemq
+description: |
+  ## Comprehensive monitoring quickstart for ActiveMQ
+  New Relic detects your ActiveMQ metrics like queues, brokers,  producers, consumers, garbage collection rate, heap memory usage and more. 
+
+  ## Why monitor ActiveMQ?
+  Apache ActiveMQ is a fault-tolerant, scalable messaging system used to build real-time data pipelines. ActiveMQ also supports replications natively, and can help build streaming applications that run inside production environments.
+
+  Monitoring ActiveMQ with our quickstart shows you performance metrics and inventory data on a real-time basis, create your own custom dashboard with charts and queries, and create alert policies.
+
+
+  ## What’s included in this quickstart?
+  New Relic’s performance monitoring provides instant observability out-of-the-box.
+
+  With this quickstart, you can:
+  - Easily monitor and identify when your ActiveMQ nodes are underperforming.
+  - View interactive dashboards show your tracked brokers, messages per sec, broker connections, memory usage, storage usage and more.
+  - Know when message processing time is about to degrade.
+
+summary: |
+  Monitor the performance metrics of your Apache ActiveMQ instances in real-time with New Relic APM.
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+title: ActiveMQ
+documentation:
+  - name: ActiveMQ integration documentation
+    description: |
+      Monitor the performance metrics of your Apache ActiveMQ instances in real-time with New Relic APM.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/jmx-monitoring-activemq
```

Comment:
This one does as well

Reply:
Hi @aswanson-nr , 
We have coordinated with @akristen from Documentation team. This docs URL is live now.
Request you to kindly do the needful.
Thank you. 😀